src/components/resource-section/add-evaluation/AddEvaluationAuth.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { InputTextarea } from 'primereact/inputtextarea';
+import { Rating } from 'primereact/rating';
+import { Card } from 'primereact/card';
+import { Button } from 'primereact/button';
+import { Toast } from 'primereact/toast';
+import styles from './AddEvaluation.module.scss';
+import useGet from '@hooks/useGet';
+import { endpoints } from '@utils/endpoints';
+
+const AddEvaluationAuth = ({ resourceId, updatesAddEvaluation }) => {
+  const {
+    data,
+    isLoading: isLoadingEvaluation,
+    isError: isErrorEvaluation,
+    mutate: updateResourceEvaluation,
+  } = useGet(endpoints('resourceEvaluation', resourceId));
```

Comment:
why a component that didn't need to call an endpoint now is calling the endpoint?

Reply:
I pushed a new version that create a new parent component, so the component doesn't call the endpoint.